README.markdown

```diff
@@ -1,522 +0,0 @@
-
-  ```puppet
-    modules => ['puppetlabs-mysql', 'puppetlabs-apache']
-  ```
-
-* `module_repos` - Modules that should be installed from github. Accepts a hash
-  where the keys indicates the directory where the module should be installed and the value points to the GitHub repo.
-
-  ```puppet
-  module_repos => { 'mysql' => 'git://github.com/puppetlabs/puppetlabs-mysql' }
-  ```
-
-* `enc_classes` - Hash of classes from our downloaded content that should be
-  applied using External Node Classifers. The key of this hash is the name of
-  a class to apply and the value is a hash of parameters that should be set
-  for that class.
-
-  ```puppet
-  enc_classes => {'mysql' => {'config_hash' => {'bind_address' => '0.0.0.0' }}}
-  ```
-
-* `manifest` - A string to pass in as a local manifest file and applied during
-  the bootstrap process. See the example manifest files in `tests/*.pp`
-  for examples on specifying full manifests.
-* `block_for_startup_script` - Whether the resource should block until its
-  startup sctipt has completed.
-* `startup_script_timeout` - Amount of time to wait before timing out when
-  blocking for a startup script.
-
-### Puppet Master and Service specification
-
-The solution allows specification of a puppet master instance with the
-`gce_instance` parameter:
-
-* `puppet_master` - Hostname of the puppet master instance that the
-  agent instance must be able to resolve.
-
-  ```puppet
-  puppet_master => 'puppet'
-  ```
-
-If this parameter is specified, then it is used as the `server` parameter in
-`puppet.conf`. If unspecified, the default of `puppet` is used.
-This parameter may be explicitly set to an empty string for a masterless instance.
-
-The solution allows specification of whether to start the puppet agent service
-with the `gce_instance` parameter:
-
-* `puppet_service` - `absent` or `present` (default `absent`)
-
-  ```puppet
-  puppet_service => present
-  ```
-
-If this parameter is specified, then the puppet service is automatically started
-on the managed instance and set to restart on boot (in `/etc/default/puppet`).
-
-### Puppet Enterprise
-
-If you choose `startupscript => 'puppet-enterprise.sh'`, you can provide data needed for the [PE installer answer file](http://docs.puppetlabs.com/pe/latest/install_answer_file_reference.html) in the `metadata` parameter. 
-The following example specifies the PE version and PE Console login details.
-
-```puppet
-   metadata             => {
-	   'pe_role'          => 'master',
-	   'pe_version'       => '3.1.0',
-	   'pe_consoleadmin'  => 'admin@example.com',
-	   'pe_consolepwd'    => 'puppetize',
-   },
-```
-This example will provision a PE Agent and will point it to your PE master.
-```puppet
-   metadata             => {
-      'pe_role'          => 'agent',
-      'pe_master'        => "[gce_instance_namevar].c.[gce_projectid].internal",
-      'pe_version'       => '3.1.0',
-   },
-```
-
-### Implementation of classification
```

Comment:
Has this functionality been removed? If so that might pose a problem for existing users, if not why remove the documentation?